public/playersBetController.js

```diff
@@ -0,0 +1,147 @@
+"use strict";
+export default class playersBetController {
+  constructor(players) {
+    this.currentPlayerTurn = 0;
+    this.players = players;
+    this.activePlayer = this.players[this.currentPlayerTurn];
```

Comment:
When this controller is initialized there are no players registered, so either defer the initialization of the bet controller to when there is a registered player or adapt the controller to work properly when there are no players active